src/reactviews/pages/ExecutionPlan/queryPlanSetup.ts

```diff
@@ -395,162 +395,179 @@ export function getCollapseExpandPaths() {
     };
 }
 
-export const save = (theme: string) => {
+// this function determines whether to use dark or light icons,
```

Comment:
What's this? 

Why don't you do something like this? 

https://github.com/microsoft/vscode-mssql/blob/516b87fa66016d4ec92f3e2325dcc378fadb03fc/src/reactviews/pages/TableDesigner/designerResultPane.tsx#L307

You can get the theme from here
https://github.com/microsoft/vscode-mssql/blob/516b87fa66016d4ec92f3e2325dcc378fadb03fc/src/reactviews/pages/TableDesigner/designerResultPane.tsx#L307

You can set this variable like this.
https://github.com/microsoft/vscode-mssql/blob/516b87fa66016d4ec92f3e2325dcc378fadb03fc/src/reactviews/pages/TableDesigner/tableDesignerStateProvider.tsx#L185-L186